build.js

```diff
@@ -0,0 +1,39 @@
+import { build } from "esbuild";
+import pkg from "./package.json" assert { type: "json" };
+
+/**
+ * we bundle jslib-media because webpack has issues resolving imports from it
+ *
+ * we bundle events because webpack tries to `require` it in the bundle which
+ * causes `Dynamic require of "events" is not supported` when jslib-media imports
+ * from mediasoup-client
+ * */
+const external = Object.keys(pkg.dependencies)
+    .concat(Object.keys(pkg.peerDependencies))
+    .filter((dep) => !dep.match(/jslib-media|events/));
+
+const sharedConfig = {
+    bundle: true,
+    external,
+    format: "esm",
+    minify: process.env.NODE_ENV === "production",
+    platform: "browser",
+    target: ["es2016"],
```

Comment:
I know this was my recommendation, but seeing we target es6 in our tsconfig, I think we should use that here as well